### PR TITLE
Remove mysql dependencies

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@MichaelKohler feel free to merge when things look OK in stage/prod

File: env-dist [hunk 1]
```diff
@@ -24,10 +24,7 @@ OIDC_OP_STORE_ACCESS_TOKEN = True
 # Mozillians
 MOZILLIANS_API_URL = https://mozillians.org/api/v2/
 # Default database setting for docker setup
-DATABASE_URL = 'mysql://remo:remo@db:3306/remo'
-# Uncomment the below DATABASE_URL setting if you're running the setup
-# locally using virtualenv
-# DATABASE_URL = 'mysql://remo:remo@localhost:3306/remo'
+DATABASE_URL = 'postgres://remo:remo@db:5432/remo'
```
Comment (MichaelKohler) [inline env-dist]: won't this break virtualenv setups? I think we should keep the comment at least

Comment (johngian) [reply env-dist]: Do we need to maintain the virtualenv setup ?

Comment (MichaelKohler) [reply env-dist]: Given that the Reps Portal will be decommissioned at some point, maybe not. However then we should also remove the docs for it.

**Comment by MichaelKohler** [comment]
https://travis-ci.org/mozilla/remo/jobs/614558195?utm_medium=notification&utm_source=github_status is failing badly

**Comment by johngian** [comment]
Yeah i think the problem is that travis has MySQL installed where we need postgres

**Pull request closed without merging**